enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -665,16 +665,13 @@ def can_redeem(self, request, enterprise_customer_uuid):
             # Determine the price for content for display purposes only.
             try:
                 content_metadata = self.subsidy_client.get_subsidy_content_data(enterprise_customer_uuid, content_key)
-                # Note that the "content_price" key is guaranteed to exist, but the value may be None.  Also, the
-                # upstream field type is a string representing a decimal dollar.
```

Comment:
> Also, the upstream field type is a string representing a decimal dollar.

This doesn't appear to be true. The `content-metadata` API in subsidy returns USD cents: https://enterprise-subsidy.edx.org/api/schema/redoc/#tag/api/operation/api_v1_content_metadata_retrieve

```json
{
    "content_uuid": "eeeb2cad-7e7b-4efa-a61e-e42b75751e70",
    "content_key": "edX+DM",
    "course_run_uuid": "d178f6d4-cd7a-4c6d-b1c8-f8099433173a",
    "course_run_key": "course-v1:edX+DM+1T2023",
    "source": "2u",
    "mode": "paid-executive-education",
    "content_price": 81800,
    "geag_variant_id": "7a3334ce-c587-4757-8c8c-d5fe9f6ed88b"
}
```

Reply:
todo for our future selves: we should add some better docstrings to the subsidy client package to make this clearer.

Reply:
Ah, for some reason I read the enterprise-subsidy code and thought it was a passthrough.  However, it seems to already contain conversion logic: https://github.com/openedx/enterprise-subsidy/blob/837fdcde75d7b0450506f637f410785a7d6f3d3e/enterprise_subsidy/apps/content_metadata/api.py#L54